backend/src/services/availabilityService.ts

```
@@ -253,88 +257,107 @@ export class AvailabilityService {
-          }
-        }
+            in: ["SCHEDULED", "CONFIRMED"],
+          },
+        },
       });
 
-      const conflicts: ConflictResult['conflicts'] = [];
+      const conflicts: ConflictResult["conflicts"] = [];
 
       for (const appointment of appointments) {
         const dayOfWeek = this.getDayOfWeek(appointment.appointmentDate);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)